docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -4,89 +4,92 @@
 
 beta::[]
 
-{anomaly-detect-cap} alerts run scheduled checks on an {anomaly-job} or a group 
-of jobs to detect anomalies with certain conditions. If an anomaly meets the 
-conditions, the alert triggers the defined action. For example, you can create 
-an alert that checks an {anomaly-job} every fifteen minutes for critical 
-anomalies and notifies you in an email. This page helps you to configure an 
-{anomaly-detect} alert. To learn more about alerts in the {stack}, refer to 
+{anomaly-detect-cap} alerting runs scheduled checks on an {anomaly-job} or a 
+group of jobs to detect anomalies with certain conditions. If an anomaly meets 
+the rules you defined, an alert is created and the associated action is 
```

Comment:
Not sure if we should use the word `rules` here, might be confusing with the new terminology, because `alert` has been renamed to `rule`.